lib/morpheme.js

```diff
@@ -125,17 +125,17 @@ export default class Morpheme {
    */
   matchingValues (feature) {
     let matches = []
+
     if (feature && this.features.hasOwnProperty(feature.type)) {
       const morphemeValue = this.features[feature.type]
-      for (const featureValue of feature.values) {
-        if (morphemeValue.values.includes(featureValue)) {
-          matches.push(featureValue)
-        }
+
+      if (feature.type !== 'full form' && morphemeValue.value === feature.value) {
```

Comment:
Do we need to have a `feature.type !== 'full form'` check here? I think it should probably work fine without.